bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/PerChannelBookieClientFactory.java

```diff
@@ -36,6 +36,6 @@
      * @return the client connected to address.
      * @throws SecurityException
      */
-    PerChannelBookieClient create(BookieSocketAddress address,
-            PerChannelBookieClientPool pcbcPool, SecurityHandlerFactory shFactory) throws SecurityException;
+    PerChannelBookieClient create(BookieSocketAddress address, PerChannelBookieClientPool pcbcPool,
```

Comment:
I would suggest using `ProtocolVersion` in this method.

Reply:
Sure